README.md

```diff
@@ -111,8 +111,8 @@ As of now, these are the supported resources:
        --productQueryParameters         Pass your customized product fetch limit
```

Comment:
Do we need to increment the versions in docs before merging those changes ?

Reply:
Do you mean for the release?

Reply:
Yes, What else could be 😄

Reply:
https://github.com/commercetools/commercetools-project-sync/pull/268/commits/7c399f4c431b04ae03dc97348384df92918bfe3c